### DISCUSSION TITLE
Please give me a hand !! Please tell me which country you like or want to visit!

### DISCUSSION THREAD

**Discussion by hiromuotsubo** [NAIST-SE/AttractingContributors#904] [2023 Attraction Contributions]
Please tell me which country you like or want to visit!
https://github.com/Tsubonee/AttractingContributors-country

好きな国 or 行ってみたい国はどこですか？
https://github.com/Tsubonee/AttractingContributors-country

![Travel plans_Flatline](https://github.com/NAIST-SE/AttractingContributors/assets/65530655/e76bfdf3-5ba5-4f58-b61b-fcc715754c7d)

**Comment by bconfessor** [comment]
I want to go to Australia someday
I sent you a pull request! 
Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md

**Comment by Gloria109** [comment]
I want to travel to Spain again ! I believe that one day my dream will come true
